docs/monitors/appmesh.md

```diff
@@ -48,20 +48,24 @@ Monitor Type: `appmesh`
 
 ## Configuration
 
+**For a list of monitor options that are common to all monitors, see [Common
+Configuration](../monitor-config.md#common-configuration).**
+
+
 | Config option | Required | Type | Description |
 | --- | --- | --- | --- |
 | `listenAddress` | no | `string` | The host/address on which to bind the UDP listener that accepts statsd datagrams (**default:** `localhost`) |
 | `listenPort` | no | `integer` | The port on which to listen for statsd messages (**default:** `8125`) |
 | `metricPrefix` | no | `string` | A prefix in metric names that needs to be removed before metric name conversion |
 
 
-
-
 ## Metrics
 
-The following table lists the metrics available for this monitor. Metrics that are marked as Included are standard metrics and are monitored by default.
+The following table lists the metrics available for this monitor.
+Metrics that are categorized as [container/host/bundled](https://docs.signalfx.com/en/latest/admin-guide/usage.html#about-custom-bundled-and-high-resolution-metrics)
```

Comment:
Bundled does not apply to these metrics. The language to be used above these tables was developed and approved a few weeks ago in discussions with Joe D. Also, it doesn't make sense to link to that doc section (here or in the table header) since Default means they are NOT custom.

Reply:
Removed the word "bundled" from this but I think we should link to those docs somewhere in here to make it very clear why the difference matters.